packages/react-native-web/src/exports/Linking/index.js

```diff
@@ -36,9 +36,9 @@ const Linking = {
   }
 };
 
-const open = url => {
+const open = (url: string, target?: '_blank') => {
   if (canUseDOM) {
-    window.location = new URL(url, window.location).toString();
+    window.open(url, target).focus();
```

Comment:
`window.open()` is different from `window.location` assignment.

They are not interchangeable. 🤔 This will open a new tab.

Reply:
This is indeed the goal